apps/staking/app/stake/[address]/NewStake.tsx

```diff
@@ -173,6 +199,12 @@ export function NewStake({ contract }: { contract: ContributionContract }) {
 
   const watchedRewardsAddress = form.watch('rewardsAddress');
 
+  useEffect(() => {
+    if (!isLoading) {
```

Comment:
Does this not need to take into account isSubmitting? Just making sure this works as expected and isn't called more than necessary?

Reply:
isLoading is if the vesting contract state is loading. so no this should in fact never run, only if someone messes up changing the code